src/libopensc/card-openpgp.c

```diff
@@ -247,8 +247,11 @@ static pgp_do_info_t	pgp34_objects[] = {	/**** OpenPGP card spec 3.4 ****/
 	{ 0x5f48, CONSTRUCTED, READ_NEVER  | WRITE_PIN3,  NULL,               sc_put_data },
 	{ 0x5f50, SIMPLE,      READ_ALWAYS | WRITE_PIN3,  sc_get_data,        sc_put_data },
 	{ 0x5f52, SIMPLE,      READ_ALWAYS | WRITE_NEVER, sc_get_data,        NULL        },
+
 	/* DO 7F21 is CONSTRUCTED in spec; we treat it as SIMPLE: no need to parse TLV */
-	{ DO_CERT, SIMPLE,      READ_ALWAYS | WRITE_PIN3,  sc_get_data,        sc_put_data },
+	/* kept for backwards compatibility (reads ID3), reading certs manually */
```

Comment:
Honestly, I did not follow the whole conversation here, but I do not understand this addition without it anyway, so... what is it for?

Reply:
that's a misleading comment, will remove it

Reply:
you did not yet, did you?

Reply:
done